source/battle.py

```diff
@@ -289,6 +322,7 @@ def encounter_text_show(
             text = '='
             text_handling.print_separator(text)
             fighting = True
+        time.sleep(1.5)
```

Comment:
If you don't put a wait time, the player won't be able to see the text appearing saying the damages, the dodge things etc... because after that, the loop restart with a 'clear' statement, that clears the screen.